clients/src/test/java/org/apache/kafka/common/utils/MockTime.java

```diff
@@ -20,13 +20,20 @@
 public class MockTime implements Time {
 
     private long nanos = 0;
+    private long autoTickMs = 0;
```

Comment:
It makes timeout testing easier if there is an automatic way to advance time. Otherwise you need to increment the clock in a separate thread from code blocking on a timeout.